Bank/IAccount.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Bank;
+
+internal interface IAccount
+{
+
+    private string _Account { get => _Account; set => _Account = value; }
+    private TypeAccount _type { get => _type; set => _type = value; }   
```

Comment:
Неправильный модификатор доступа у члена интерфейс. Все члены интерфейса должны быть публичными. Модификатор доступа у членов интерфейса не ставится.

Названия свойств кривые.

Reply:
Некоректно используете реализацию члена на уровне интерфейс. Не надо так.
```suggestion
    string Account { get; set; }
    TypeAccount Type { get; set; }   
```